### DISCUSSION TITLE
SQL Editor Favorite queries is not sync in different OS/device

### DISCUSSION THREAD

**Discussion by aleja-lass** [supabase/supabase#2172] [Questions] [answered]
Saved and Favorites queries in the Supabase SQL Editor in Arch Linux (Browser: Brave) does not show up in Windows 10 (Browser: Chrome) and vise versa.

**Comment by steve-chavez** [comment, marked as answer]
hey @aleja-lass,

That is true, right now the queries are only saved in local storage. Please raise an [issue](https://github.com/supabase/supabase/issues/) with the feature request.
